specification/compiler/model/metamodel.ts

```diff
@@ -189,6 +207,11 @@ export class Request extends BaseType {
   body?: ValueBody | PropertiesBody
   behaviors?: Implements[]
   attachedBehaviors?: string[]
+
+  since?: string
+  restSpecName?: string;
```

Comment:
This should be `endpointName`. But I don't think this information is useful since we already have the request type identified in endpoints.

Reply:
I've moved all properties to the endpoint now and throw if they differ from the spec